samples/MvcSample.Web/HomeController.cs

```diff
@@ -113,6 +113,22 @@ public ActionResult SaveUser(User user)
         }
 
         /// <summary>
+        /// Action that shows binding of form values to <see cref="IFormCollection"/>.
+        /// </summary>
+        public ActionResult PostForm(IFormCollection form)
+        {
+            if (!ModelState.IsValid)
```

Comment:
I think there would be no validation for the model if its `IFormCollection` here(as its just raw collection and not some POCO object with data annotations attributes for example)?